src/idex/common.js

```diff
@@ -108,25 +108,27 @@ export function makeIdentityResolver (config, calls, cache) {
     }
 
     const responseReceived = (additionalParams, successCallback) => {
-      return response => {
+      return (responseText, response) => {
         let responseObj = {}
-        if (response) {
+        if (responseText) {
           try {
-            responseObj = JSON.parse(response)
+            responseObj = JSON.parse(responseText)
           } catch (ex) {
             console.error('Error parsing response', ex)
             fromError('IdentityResolverParser', ex)
           }
         }
-        cache.set(additionalParams, responseObj)
+
+        const expiresAt = responseExpires(response)
```

Comment:
should it be?
```suggestion
        const expiresAt = responseExpires(responseObj)
```

Reply:
We need to extract it from the headers. ResponseText is only the body